packages/evolution-backend/src/services/audits/auditChecks/checks/__tests__/interview/testHelper.ts

```
@@ -0,0 +1,32 @@
+import { InterviewAuditCheckContext } from '../../../AuditCheckContexts';
+import { InterviewParadata } from 'evolution-common/lib/services/baseObjects/interview/InterviewParadata';
+
+export const createMockInterview = (overrides: Partial<Interview> = {}, validUuid = uuidV4(), validId = 123) => {
+    return {
+        _uuid: validUuid,
+        get uuid() { return this._uuid; },
+        id: validId,
+        paradata: new InterviewParadata({ languages: [{ language: 'en' }] }),
+        ...overrides
+    } as Interview;
+};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)